DEVELOP.md

```diff
@@ -0,0 +1,54 @@
+# Local development guide
+This guide is made up of two parts, the first being on **web server setup** and the second being about **Raspberry Pi setup**.
+
+## Web server setup
+### Downloads
+
+1. [Install PostgreSQL](https://www.postgresql.org/download/).
+2. [Install NodeJS](https://nodejs.org/en/).
```

Comment:
On Linux these can be installed via the command:
`sudo apt-get install postgresql postgresql-contrib nodejs`

`npm install` works the same on Linux of course. 

To run `setup.sql` on Linux you'll need to execute `psql -f setup.sql` instead (see [post on StackOverflow](https://stackoverflow.com/questions/9736085/run-a-postgresql-sql-file-using-command-line-arguments). You may want to execute this command as postgres (see [DigitalOcean tutorial](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-postgresql-on-ubuntu-16-04))).